.rubocop.yml

```diff
@@ -73,3 +73,8 @@ Style/OptionalBooleanParameter:
 # it sometimes make code harder to read
 Style/GuardClause:
   Enabled: false
+
+# disable missing ending range as it is not supported in ruby 2.7
+# TODO: is it bug in rubocop that with target ruby version 2.7 it stil do it?
```

Comment:
Interesting question.
The current source for the cop thinks endless ranges should work even since Ruby 2.6:
https://github.com/rubocop/rubocop/blob/751898494b694f0520d1739b877e1add94c71366/lib/rubocop/cop/style/slicing_with_range.rb

Reply:
Ah, it is my fault. ruby in SLE15 is 2.5 and not 2.7.
```
jreidinger@jreidinger:~/prace/yast/registration> ruby --version
ruby 2.5.9p229 (2021-04-05 revision 67939) [x86_64-linux-gnu]
```